vega_sim/wallet/vega_wallet.py

```diff
@@ -123,7 +126,6 @@ def generate_keypair(
             json=req,
         )
         response.raise_for_status()
-        return list(response.json()["key"]["pub"])
```

Comment:
Why stop returning the key?

Reply:
Just the output didn't seem to be used in any of the calls in `VegaWallet` or `SlimWallet`. Tbh I didn't check through the entire codebase so have added back in just in case!